DOCS/man/vf.rst

```diff
@@ -559,6 +559,10 @@ Available mpv-only filters are:
         By default, this uses the special value ``auto``, which sets the option
         to the number of detected logical CPU cores.
 
+    ``user-data``
+        Optional arbitrary data that is passed to the script. Default to empty
```

Comment:
It is fine, but it's more like arbitrary string, cannot really pass data with null bytes in it.

Reply:
Fixed